tests/integration/udf/test_vector_cosine_similarity.py

```diff
@@ -0,0 +1,34 @@
+"""
+Test vector cosine similarity
+"""
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize("source_type", ["snowflake"], indirect=True)
```

Comment:
Should spark be covered in this test as well?

Reply:
oops forgot to add that in here after adding in the spark UDF. will add too